doc/esmvalcore/preprocessor.rst

```diff
@@ -905,6 +906,21 @@ set manually by the user in recipe.
 See also :func:`esmvalcore.preprocessor.regrid_time`.
 
 
+.. _timeseries_filter:
+
+``timeseries_filter``
+---------------------
+
+This function allows the user to apply a filter to the timeseries data. This filter may be
+of the user's choice (currently only the ``low-pass`` Lanczos filter is implemented); the
+implementation is inspired by this `iris example
+<https://scitools.org.uk/iris/docs/latest/examples/General/
+SOI_filtering.html?highlight=running%20mean>`_ and uses aggregation via a
+`rolling window <https://scitools.org.uk/iris/docs/v2.0/iris/iris/cube.html#iris.cube.Cube.rolling_window>`_.
+
```

Comment:
Can you make this a bit nicer? I am missing the Parameters and Examples sections, see e.g. https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/preprocessor.html#climate-statistics